models/gpt2/config.py

```diff
@@ -18,4 +18,4 @@
 # T - decode step size
 def get_config():
     config = collections.namedtuple('Config', ['B', 'K', 'S', 'T'])
-    return config(1, 8, 64, 1)
```

Comment:
Hi @rsuderman , I just realized that export.py does not longer generate MLIR that can be used in the iOS on-device fine-tune app. It requires B=1 here.

Reply:
I wanted to keep the B=4 as a default to guarantee batching works correctly. We should add a flag here:
https://github.com/iree-org/iree-jax/blob/main/models/gpt2/config.py#L13

To override batching on export / testing for this kind of usecase.

Reply:
I agree. I filed https://github.com/iree-org/iree-jax/pull/68 Please take a look.